chaoscenter/src/database_model.py

```diff
@@ -0,0 +1,25 @@
+# Copyright 2025 Canonical
+# See LICENSE file for licensing details.
+
+"""Database Config class."""
+
+import dataclasses
+from typing import Optional
+
+
+@dataclasses.dataclass(init=False)
+class DatabaseConfig:
+    """Model for database client relation databag."""
+
```

Comment:
```suggestion
    """Model for database client relation databag.
    
    We need this because the mongo charm lib doesn't expose a typed data structure, so we wrap their API with this instead of passing around raw dicts.
    """

```